### PR TITLE
Fix dependencies vuepress theme v64

### PR DESCRIPTION
**Pull request by Amndeep7**

### CODE REVIEW

**Review by wdower — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -16,8 +16,6 @@
   },
   "devDependencies": {
     "@vuepress/bundler-vite": "2.0.0-rc.19",
-    "@vuepress/plugin-markdown-image": "2.0.0-rc.66",
```
Comment (aaronlippold) [inline package.json:19]: Are these suposed to be removed or was there a change we expected to see in this file?

**Review by aaronlippold — CHANGES_REQUESTED** [review]
Perhaps a chage? see question.

File: package.json (hunk 1 above)
Comment (Amndeep7) [reply package.json:19]: As far as I can tell, the theme has bundled it internally and no longer needs you to install these plugins yourself.  You can see that the functionality still works due to graphics still being displayed properly: 
image

image
Here is the package-lock file which shows that the plugin is still installed even after I removed it from our package.json file.

**Comment by aaronlippold** [comment]
You should also let george know that this plugin is auto installed but just has to be enabled in the theme settings, as I think he and Dan work trying to build their own method but they could just use this.

**Pull request merged**